### DISCUSSION TITLE
max entity, max block type , detailed config

### DISCUSSION THREAD

**Discussion by ca9an** [GriefPrevention/GriefPrevention#827] [Q&A] [answered]
when i editing messages.yml i notice these lines

ClaimTooSmallForEntities:
    Text: This claim isn't big enough for that.  Try enlarging it.
  TooManyEntitiesInClaim:
    Text: This claim has too many entities already.  Try enlarging the claim or removing
      some animals, monsters, paintings, or minecarts.
so i think there might be a setting edits max entity type  and maybe limiting some blocks like seeds or cactuses to prevent players from building 10 floor farms. but i cant find these setting in config.
are there any detailed configs for these plugin for me to edit these settings?

**Comment by RoboMWM** [comment, marked as answer]
The value is not configurable - BigScary intended it to be this way.

These restrictions only apply to creative worlds though, and it sounds like your gamemode is not creative.
